### DISCUSSION TITLE
Tapping outside of PR Description edit window discards all changes without warning

### DISCUSSION THREAD

**Discussion by DrJacobHolden** [community/community#134913] [Mobile]
And there's no recovery. Just spent 15 minutes torturously editing my description only to have it all wiped out. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.171.1 (137945355)
Device: iPad16,3 (iOS 17.5.1)
</details>

**Comment by alcere** [comment]
Hi @DrJacobHolden! I am sorry for the troubles! I will create a ticket and the team will investigate it as soon as possible
